tests/data_context/test_data_context_test_yaml_config.py

```diff
@@ -1343,3 +1344,31 @@ def test_golden_path_runtime_data_connector_and_inferred_data_connector_pandas_d
     # Confirm that logs do not contain any exceptions or invalid messages
     assert not usage_stats_exceptions_exist(messages=caplog.messages)
     assert not usage_stats_invalid_messages_exist(messages=caplog.messages)
+
+
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
+def test_rule_based_profiler_integration(
+    mock_emit, caplog, empty_data_context_stats_enabled, test_df, tmp_path_factory
+):
+    context = empty_data_context_stats_enabled
+    yaml_config = """
+    name: my_profiler
+    class_name: RuleBasedProfiler
+    module_name: great_expectations.rule_based_profiler
+    config_version: 1.0
+    variables:
```

Comment:
I think here you can use a more complex config (e.g. with variables and some rules) unless that is tested elsewhere.

Reply:
done!